backend/src/main/java/ecse428/backend/controller/CustomerController.java

```diff
@@ -51,4 +51,20 @@ public ResponseEntity<?> checkUserCredentials(@Valid @RequestBody CustomerDto cu
             return ResponseEntity.badRequest().body(ex.getMessage());
         }
     }
```

Comment:
Instead of Requesting Body customerDto and Request Param restrictions
You could simply request Body with JSON containing email ,and list of restrictions.

RequestParam for a list of string isn't best practice